src/main/resources/templates/index.html

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Feel free to delete this file to avoid conflict later, since I have designed a home page